x-pack/legacy/plugins/uptime/server/graphql/unsigned_int_scalar/resolvers.ts

```diff
@@ -4,34 +4,48 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { GraphQLScalarType, Kind } from 'graphql';
+import { GraphQLScalarType, Kind, ValueNode } from 'graphql';
 import { UMServerLibs } from '../../lib/lib';
 import { CreateUMGraphQLResolvers } from '../types';
 
+export const serialize = (value: any): number => {
+  // `parseInt` will yield `2019` for a value such as "2019-07-08T16:59:09.796Z"
+  if (isNaN(Number(value))) {
+    return Date.parse(value).valueOf();
+  }
+  return parseInt(value, 10);
+};
+
+export const parseValue = (value: any) => {
+  const parsed = parseInt(value, 10);
+  if (parsed < 0) {
+    return null;
+  }
+  return parseInt(value, 10);
```

Comment:
`return parsed;` ?

Reply:
Indeed. This shouldn't have passed the original review where it was added.

Fixed in https://github.com/elastic/kibana/pull/40526/commits/b120855d8e7a5f1a84521793e5bc2e69a6f66b35.